website/docs/r/aws_auth_backend_client.html.md

```diff
@@ -38,6 +38,18 @@ resource "vault_aws_auth_backend_client" "example" {
 }
 ```
 
+```hcl
+resource "vault_auth_backend" "example" {
```

Comment:
Ah nice, good call on an additional example! I am going to follow this for GCP as well :)